code/modules/paperwork/paper.dm

```diff
@@ -51,6 +51,17 @@
 	if(!mapload && persist_on_init)
 		SSpersistence.track_value(src, /decl/persistence_handler/paper)
 
+/**Creates a complete copy of this object */
+/obj/item/paper/proc/Clone()
+	var/obj/item/paper/P = new(loc, material?.type, info, name)
+	P.fields             = fields
+	P.last_modified_ckey = last_modified_ckey
+	P.rigged             = rigged
+	P.is_crumpled        = is_crumpled
+	P.stamp_text         = stamp_text
+	P.applied_stamps     = LAZYLEN(applied_stamps)? applied_stamps.Copy() : null
+	P.metadata           = LAZYLEN(metadata)?       metadata.Copy()       : null
+
 /obj/item/paper/GetCloneArgs()
 	return list(null, material, info, name)
```

Comment:
As far as I know, this Clone definition is redundant and folded into the datum-level cloning procs.
```suggestion
/obj/item/paper/GetCloneArgs()
	return list(null, material?.type, info, name)
```